### PR TITLE
README.md - clarify IKS01A2 vs. 02A1

### PR DESCRIPTION
**Pull request by JanneKiiskila**
Fix for the issue #9 .

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -6,8 +6,10 @@
 This example project mainly targets the STM32L496AG-DISCOVERY development kit
 [P-L496G-CELL02](https://www.st.com/en/evaluation-tools/p-l496g-cell02.html)
 along with the optional
-[X-NUCLEO-IKS02A1](https://www.st.com/en/ecosystems/x-nucleo-iks02a1.html)
-sensor board.
+[X-NUCLEO-IKS01A2](https://www.st.com/en/ecosystems/x-nucleo-iks02a1.html#overview)
```
Comment (kFYatek) [inline README.md]: I think it would be appropriate to also update the URL for the link accordingly.

**Review by kFYatek — CHANGES_REQUESTED** [review]

**Comment by JanneKiiskila** [comment]
@kFYatek  https://github.com/AVSystem/Anjay-mbedos-client/pull/11 - probably supercedes this, so probably we could close this one and apply similar changes either as part of (additional commit via #11 ) or do an additional new PR.

File: README.md (hunk 1 above)
Comment (JanneKiiskila) [reply README.md]: It's been fixed now.

**Comment by kFYatek** [comment]
> @kFYatek #11 - probably supercedes this, so probably we could close this one and apply similar changes either as part of (additional commit via #11 ) or do an additional new PR.

OK, I'll look at #11 first then.

**Pull request closed without merging**